src/main/scala/pl/project13/scala/akka/raft/Candidate.scala

```diff
@@ -32,28 +32,55 @@ private[raft] trait Candidate {
         stay() using includingThisVote.withVoteFor(m.currentTerm, m.clusterSelf)
       }
 
-    case Event(msg: RequestVote, m: ElectionMeta) if m.canVoteIn(msg.term) =>
-      sender ! VoteCandidate(m.currentTerm)
-      stay() using m.withVoteFor(msg.term, candidate())
+		case Event(msg: RequestVote, m: ElectionMeta) if msg.term < m.currentTerm =>
```

Comment:
Minor nit: it looks like the rest of the source code uses two spaces for indentation, whereas this pull request has tabs